pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -110,6 +134,62 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dm *v1alpha1.DomainMappi
 	return err
 }
 
+func autoTLSEnabled(ctx context.Context, dm *v1alpha1.DomainMapping) bool {
+	if !config.FromContext(ctx).Network.AutoTLS {
+		return false
+	}
+	logger := logging.FromContext(ctx)
```

Comment:
Move logger in the `if` since it's unused otherwise.

Reply:
Done